src/test/java/build/buildfarm/server/BuildFarmServerTest.java

```diff
@@ -306,15 +309,20 @@ public void cancellingExecutingOperationFailsPoll()
   @Test(expected = StatusRuntimeException.class)
   public void actionWithExcessiveTimeoutFailsValidation()
       throws RetryException, InterruptedException, InvalidProtocolBufferException {
-    Action actionWithExcessiveTimeout = createSimpleAction().toBuilder()
-        .setTimeout(Duration.newBuilder().setSeconds(9000))
-        .build();
+    Digest actionDigestWithExcessiveTimeout = createAction(
+        (action) -> action.toBuilder()
+            .setTimeout(Duration.newBuilder().setSeconds(9000))
+            .build());
+
+    executeAction(actionDigestWithExcessiveTimeout);
+  }
 
-    executeAction(actionWithExcessiveTimeout);
+  private Digest createSimpleAction() throws RetryException, InterruptedException {
+    return createAction((action) -> action);
   }
 
-  private Action createSimpleAction() throws RetryException, InterruptedException {
-    DigestUtil digestUtil = new DigestUtil(DigestUtil.HashFunction.SHA256);
+  private Digest createAction(Function<Action, Action> onAction) throws RetryException, InterruptedException {
```

Comment:
Consider refactoring to accepting the action builder as a parameter instead and using it as a "starter" in line 332, instead of Action.newBuilder().